source/index.html.md

```diff
@@ -1301,12 +1302,63 @@ POST https://gateway.clearhaus.com/authorizations/:id/captures
     <div class="type">May not be all digits, all same character, or all sequential characters (e.g. "abc")</div>
     <div class="type">Optional</div>
   </dd>
+  <dt>travel
+    <span class="type">dictionary</span>
+  </dt>
+  <dd>
+    See <a href="#travel-data">Travel data</a>.
+    <div class="type">Optional</div>
+  </dd>
 </dl>
 
 <p class="alert alert-info">
-<b>Notice:</b> A capture cannot be made if the authorization is 180 days old.
+<b>Notice:</b> A capture cannot be made if the authorization is 180 days old or more.
 </p>
 
+
+##### Travel data
+
+At most one type of travel data can be supplied for a capture; if `travel` is
+supplied, it must include exactly one of `travel[car][...]`,
+`travel[flight][...]`, or `travel[lodging][...]`.
+
+For service type `[car]` (rental), the following parameter is relevant.
+
+<dl class="dl-vertical">
+  <dt>
+    travel[car][pick_up_date]
+    <span class="type">20[0-9]{2}-[0-9]{2}-[0-9]{2} (YYYY-MM-DD)</span>
```

Comment:
The same suggestion applies to the other two dates below.

Reply:
It's not too important to me, but I'm always just defaulting to the [suggestion of ISO 8601](https://en.wikipedia.org/wiki/ISO_8601#Calendar_dates) — that is, to always have the dashes to avoid confusion. In this case there's no potential confusion, so that voids the argument, and we're back to "if you always have dashes, you can always stay consistent", and that's definitely not a very strong argument, so ... 🤷‍♂️

Reply:
If we decide to remove the dashes, let me also bring up the suggestion to use the regex `[0-9]{8}`. I assume we will have a filter checking that it's a valid date.

I think https://en.wikipedia.org/wiki/Law_of_triviality applies to "dashes or not" :smile:

Reply:
After a not very long discussion we have agreed on keeping the dashes :slightly_smiling_face: :game_die: